app-drawer/app-drawer.html

```diff
@@ -535,6 +538,39 @@
         this.fire('app-drawer-overlay-changed', { overlay: this.opened && !this.persistent });
       },
 
+      _openedChanged: function(opened) {
+        var evtName = this.opened ? 'opened' : 'closed';
+
+        this.fire('app-drawer-' + evtName);
```

Comment:
Instead of making new events, can you rely on app-drawer's `opened-changed` event, then read the app-drawer's `opened` property (`opened-changed` is provided since there's `notify: true`, see https://www.polymer-project.org/1.0/docs/devguide/data-binding.html#change-notification-protocol)